package.json

```diff
@@ -8,6 +8,7 @@
     "email": "todd@twolfson.com",
     "url": "http://twolfson.com/"
   },
+  "license": "Unlicense",
```

Comment:
Can you relocate this to the existing `licenses` position and remove `licenses`?